pump/config.go

```diff
@@ -65,8 +65,8 @@ type Config struct {
 	EtcdDialTimeout   time.Duration
 	DataDir           string `toml:"data-dir" json:"data-dir"`
 	HeartbeatInterval int    `toml:"heartbeat-interval" json:"heartbeat-interval"`
-	// pump only stores binlog events whose ts >= current time - GC(day)
-	GC       int             `toml:"gc" json:"gc"`
```

Comment:
I think changing the type is a breaking change for toml marshaling and unmarshaling?
but ok for the command line.

maybe you can do this for compatibility:
```
type GC time.Duration
... impl https://pkg.go.dev/encoding?tab=doc#TextUnmarshaler interface for `GC` type
for handling both types
``

or you may keep GC int and add another config item for compatibility

Reply:
Yes, it is a breaking change for toml unmarshaling... I prefer the first solution. If we add another field I think it will be more confusing for the users.

Reply:
Prefer first too, you can try it.